subprojects/maven/src/main/java/org/gradle/api/publish/maven/internal/publication/DefaultMavenPublication.java

```diff
@@ -358,15 +359,21 @@ public String getUri() {
         };
     }
 
-    /*
-      When the artifacts declared in a component are modified for publishing (name/classifier/extension),
-      then the Maven publication no longer represents the underlying java component.
-      Instead of publishing incorrect metadata, we fail any attempt to publish the module metadata.
+    @Nullable
+    @Override
+    public ImmutableAttributes getAttributes() {
+        return null;
+    }
 
-      In the long term, we will likely prevent any modification of artifacts added from a component.
-      Instead, we will make it easier to modify the component(s) produced by a project, allowing the published
-      metadata to accurately reflect the local component metadata.
-     */
+    /*
+          When the artifacts declared in a component are modified for publishing (name/classifier/extension),
+          then the Maven publication no longer represents the underlying java component.
+          Instead of publishing incorrect metadata, we fail any attempt to publish the module metadata.
+
+          In the long term, we will likely prevent any modification of artifacts added from a component.
+          Instead, we will make it easier to modify the component(s) produced by a project, allowing the published
+          metadata to accurately reflect the local component metadata.
+         */
```

Comment:
The formatting of this comment seems to be off.

Reply:
I didn't change it, but I reformatted anyway.